sdk/schemaregistry/Microsoft.Azure.Data.SchemaRegistry.ApacheAvro/README.md

```diff
@@ -115,19 +115,19 @@ Console.WriteLine(eventData.EventBody);
 
 Similarly, to decode:
 ```C# Snippet:SchemaRegistryAvroDecodeEventDataGenerics
-Employee deserialized = (Employee) await encoder.DecodeMessageDataAsync<Employee>(eventData);
+Employee deserialized = (Employee) await serializer.DeserializeAsync<Employee>(eventData);
 Console.WriteLine(deserialized.Age);
 Console.WriteLine(deserialized.Name);
 ```
 
 ### Encode and decode data using `MessageWithMetadata` directly
```

Comment:
```suggestion
### Encode and decode data using `BinaryContent` directly
```

Reply:
And do you want to change all the wording to "serialize and deserialize" instead of "encode and decode" across this entire file?